components/PartnerBenefits.tsx

```diff
@@ -0,0 +1,39 @@
+import Link from "next/link";
+
+interface IButtonProps {
+  title: string;
+  linkPath: string;
+}
+
+export default function PartnerBenefits() {
+  function LinkButton({ title, linkPath }: IButtonProps) {
+    return (
+      <Link
+        className="rounded-xl bg-cesium-orange px-6 py-4 text-xl font-bold text-white"
+        href={linkPath}
+      >
+        {title}
+      </Link>
+    );
+  }
```

Comment:
Also, make it as a component inside the `components/` folder because we can reuse

Reply:
Its ready with the last commit, i also changed the h1 to h2